doc/nrf/ug_memfault.rst

```diff
@@ -0,0 +1,207 @@
+.. _ug_memfault:
+
+Memfault integration
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+`Memfault`_ is a cloud-based web application with |NCS| compatibility that monitors devices and allows you to debug issues remotely.
+ Memfault integration provides the following features to |NCS|:
+
+* `Remote debugging <Memfault debugging_>`_ - Memfault issues can be efficiently resolved with automatic detection, alerts, deduplication, and actionable insights sent through the cloud.
+* `Continuous monitoring <Memfault monitoring_>`_ - Monitors device and fleet-level metrics like connectivity and low power state in real-time dashboards.
+  There are 15 prepopulated metrics shown for your devices.
```

Comment:
Maybe rephrase this to not reference the number of prepoulated metrics (`15`). To reduce maintainance between versions.